src/Entity/DTO/CourseDTO.php

```diff
@@ -15,199 +16,186 @@
 class CourseDTO
 {
     /**
-     * @var int
      * @IS\Id
      * @IS\Expose
      * @IS\Type("integer")
      */
-    public $id;
+    public int $id;
 
     /**
-     * @var string
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $title;
+    public string $title;
```

Comment:
title is nullable, should be `?string $title`.

Reply:
✔️